packages/app-project/src/screens/ProjectHomePage/components/Hero/HeroContainer.spec.js

```diff
@@ -0,0 +1,166 @@
+    let componentWrapper
+
+    before(function () {
+      scope = nock('https://panoptes-staging.zooniverse.org/api')
+        .get('/translations')
+        .query(true)
+        .reply(200, {
+          translations: TRANSLATIONS
+        })
+        .get('/workflows')
+        .query(true)
+        .reply(200, {
+          workflows: WORKFLOWS
+        })
+      wrapper = shallow(<HeroContainer activeWorkflows={['1']} />)
+      componentWrapper = wrapper.find(Hero)
+    })
+
+    after(function () {
+      nock.cleanAll()
+    })
+
+    it('should render without crashing', function () {
+      expect(wrapper).to.be.ok()
+    })
+
+    it('should render the `Hero` component', function () {
+      expect(componentWrapper).to.have.lengthOf(1)
+    })
+  })
+
+  describe('loading state', function () {
+    let scope
+    let wrapper
+    let componentWrapper
+
+    before(function () {
+      scope = nock('https://panoptes-staging.zooniverse.org/api')
+        .get('/translations')
+        .query(true)
+        .reply(200, {
+          translations: TRANSLATIONS
+        })
+        .get('/workflows')
+        .query(true)
+        .reply(200, {
+          workflows: WORKFLOWS
+        })
+    })
+
+    after(function () {
+      nock.cleanAll()
+    })
+
+    it('should pass down the correct props', function () {
+      wrapper = shallow(<HeroContainer activeWorkflows={['1']} />)
+      componentWrapper = wrapper.find(Hero)
+      expect(componentWrapper.prop('workflows')).to.deep.equal({
+        loading: 'loading',
+        data: []
+      })
+    })
+  })
+
+  describe('success state', function () {
+    let scope
+    let wrapper
+    let componentWrapper
+    let fetchWorkflowsSpy
+
+    before(function () {
+      scope = nock('https://panoptes-staging.zooniverse.org/api')
+        .get('/translations')
+        .query(true)
+        .reply(200, {
+          translations: TRANSLATIONS
+        })
+        .get('/workflows')
+        .query(true)
+        .reply(200, {
+          workflows: WORKFLOWS
+        })
+      fetchWorkflowsSpy = sinon.spy(HeroContainer.prototype, 'fetchWorkflows')
+    })
+
+    after(function () {
+      nock.cleanAll()
+      fetchWorkflowsSpy.restore()
+    })
+
+    it('should pass down the correct props', async function () {
+      wrapper = shallow(<HeroContainer activeWorkflows={['1']} />)
+      await fetchWorkflowsSpy.returnValues[0]
+      componentWrapper = wrapper.find(Hero)
+      expect(componentWrapper.prop('workflows')).to.deep.equal({
+        loading: 'success',
+        data: [
+          { completeness: 0.4, default: true, id: '1', displayName: 'Foo' },
+        ]
+      })
+    })
+  })
+
+  describe('error state', function () {
+    let scope
+    let wrapper
+    let componentWrapper
+    let fetchWorkflowsSpy
+
+    before(function () {
+      scope = nock('https://panoptes-staging.zooniverse.org/api')
+        .get('/translations')
+        .query(true)
+        .reply(200, {
```

Comment:
Non-blocking, but if returning an error, just for consistency in mocking, this probably shouldn't be a 200 status.

Reply:
Only the workflows endpoint is returning an error in this test, but both requests need to be successful in order to show the content (as currently written)